pkg/cloud/services/ec2/launchtemplate_test.go

```diff
@@ -1190,6 +1240,9 @@ func TestCreateLaunchTemplateVersion(t *testing.T) {
 			g.Expect(err).NotTo(HaveOccurred())
 
 			ms, err := setupMachinePoolScope(client, cs)
+			if tc.useCapacityBlocks {
+				ms, err = setupCapacityBlocksMachinePoolScope(client, cs)
+			}
```

Comment:
Should be if/else instead of overwriting `ms` with something else. Or pass an argument to `setupMachinePoolScope` if the scopes are very similar?

Reply:
No, two different methods we are calling. Created ms variable to make if/else.